File: Backend_DotNet/WheelShare_dotNet/WheelShare_dotNet/DTOs/SavedRideDto.cs
namespace WheelShare_dotNet.DTOs
{
    public class SavedRideDto
    {
        public string RideName { get; set; } = string.Empty;
        public string PickupAddress { get; set; } = string.Empty;
        public double PickupLatitude { get; set; }
        public double PickupLongitude { get; set; }
        public string DropAddress { get; set; } = string.Empty;
        public double DropLatitude { get; set; }
        public double DropLongitude { get; set; }
    }
}

File: Backend_Springboot/cab-booking/src/main/java/com/wheelshare/cabbooking/dto/SavedRideDto.java
package com.wheelshare.cabbooking.dto;

import lombok.Data;

@Data
public class SavedRideDto {
    private String rideName = "";
    private String pickupAddress = "";
    private Double pickupLatitude;
    private Double pickupLongitude;
    private String dropAddress = "";
    private Double dropLatitude;
    private Double dropLongitude;
}
